x-pack/plugins/observability_solution/ux/kibana.jsonc

```diff
@@ -24,19 +24,13 @@
       "usageCollection",
       "taskManager",
       "actions",
-      "alerts",
```

Comment:
`alerts` as a plugin doesn't exist. Apparently the CI does not check `optionalPlugins` inside `kibana.jsonc`, only `requiredPlugins`.